src/webview/topic-config-form.html

```diff
@@ -198,7 +224,7 @@ <h1>Configure Topic: <span data-text="this.topicName()"></span></h1>
       .button.ghost {
         background-color: transparent;
         border: none;
-        color: var(--vscode-button-secondaryForeground);
+        color: var(--vscode-descriptionForeground);
```

Comment:
Threw this in bc I noticed the VSCode Light theme doesn't show the secondary foreground color on this button